content/applications/sales/sales/products_prices/loyalty_discount.rst

```diff
@@ -93,6 +99,8 @@ The different :guilabel:`Program Types` available on the program form are:
   customer.
 - :guilabel:`Buy X Get Y`: For every X item bought, the customer is granted 1 credit. After
   accumulating a specified amount of credits, the customer can trade them in to receive Y item.
+- :guilabel:`Next Order Coupons`: Generate and share single-use coupon codes that grant access to
```

Comment:
```suggestion
- :guilabel:`Next Order Coupons`: generate and share single-use coupon codes that grant access to
```
If this is a bulleted list, I believe we do the first word after the colon lowercase for all of these.